### PR TITLE
Global burn rate

### PR DESCRIPTION
**Pull request by sntwo**
Added burn rate to modify the rate burning rounds damage objects.  Changeable in game in info tweak.

### CODE REVIEW

File: code/code/misc/statistics.cc [hunk 1]
```diff
@@ -207,6 +209,10 @@ int init_game_stats(void)
       vlogf(LOG_BUG, "bad value for rate of change of equipment load potential");
     }
 
+    if (fscanf(fp, "%lf\n", &stats.burnrate) != 1) {
```
Comment (cizra) [inline code/code/misc/statistics.cc]: Why not store it in DB? You could reuse the `configuration` table, it's a key-value mapping -- or even better, create a new table for all stats. See `code/sys/migrations.cc` for a comfy way to do that.

File: code/code/misc/statistics.cc [hunk 2]
```diff
@@ -69,6 +69,8 @@ int init_game_stats(void)
   stats.global_lp_target_changerate = 0.0;  // This is how fast the load potential changes.
   stats.max_exist = 1.2;  // this affects the MAX number of a thing allowed
 
+  stats.burnrate = 0.4;  // this affects the rate at which things burn
```
Comment (cizra) [inline code/code/misc/statistics.cc]: Shouldn't the default be 1?

File: code/code/misc/statistics.h [hunk 3]
```diff
@@ -107,6 +109,7 @@ class GameStats {
       piety_gained(0.0),
       piety_gained_attempts(0),
       equip(0.0),
+      burnrate(0.0),
```
Comment (cizra) [inline code/code/misc/statistics.h]: uhhh.. 1?

File: code/code/misc/statistics.cc (hunk 1 above)
Comment (sntwo) [reply code/code/misc/statistics.cc]: This makes sense.  I was trying to keep my first commit simple so I just copied the stats.load format.  There are many magic constants scattered around that could be pulled into this.  I'll look into it.

**Pull request merged**